src/components/IndividualArticleCard.jsx

```diff
@@ -0,0 +1,34 @@
+import { getArticleById } from "../utils/api";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
+const IndividualArticleCard= () => {
+  const { article_id } = useParams();
+  const [article, setArticle] = useState({});
+
```

Comment:
We need some loading feedback for a better UX - let the user know that something is happening/data is being requested 💡